docs/source/model_parallel.rst

```diff
@@ -0,0 +1,11 @@
+.. role:: hidden
+    :class: hidden-section
+
+Distributed RPC Framework - torch.distributed.autograd and torch.distributed.rpc
```

Comment:
Only `Distributed RPC Framework` should be sufficient, otherwise the entire title is too long for the title.